content/spin/v2/triggers.md

```diff
@@ -69,11 +130,11 @@ These ways of writing components achieve the same thing, so which should you cho
 Named components have the following advantages:
 
 * Reuse. If you want two triggers to behave in the same way, they can refer to the same named component. Remember this means they are not just handled by the same Wasm file, but with the same settings.
-* Named. If an error occurs, Spin can tell your the name of the component where the error happened. With inline components, Spin has to synthesize a name. This isn't a big deal in single-component apps, but make diagnostics harder in larger apps.
+* Named. If an error occurs, Spin can tell you the name of the component where the error happened. With inline components, Spin has to synthesize a name. This isn't a big deal in single-component apps but makes diagnostics harder in larger apps.
 
 Inline components have the following advantages:
 
 * Compact, especially when using inline table syntax.
 * One place to look. Both the trigger event and the handling details are always in the same piece of TOML.
 
-If you are not sure, or are not experienced, we recommend using named components at first, and adopting inline components as and when you find cases where you prefer them.
+If you are not sure or are not experienced, we recommend using named components at first and adopting inline components as and when you find cases where you prefer them.
```

Comment:
I found this easier to parse with the commas.

Reply:
Updates in next push.